turbopy/core.py

```diff
@@ -544,7 +544,7 @@ class SimulationClock:
     """
 
     def __init__(self, owner: Simulation, clock_data: dict):
-        self.owner = owner
+        self._owner = owner
         self.clock_data = clock_data
```

Comment:
I'm not sure why I called this `clock_data` instead of `input_data`. Let's also change this to `_input_data` for the sake of consistency.
```suggestion
        self._input_data = clock_data
```

Reply:
Should I change `grid_data` in the `Grid` class to `input_data` too